lib/api/BackbeatAPI.js

```diff
@@ -236,7 +239,9 @@ class BackbeatAPI {
     getHealthcheck(details, cb) {
         return this._healthcheck.getHealthcheck((err, data) => {
             if (err) {
-                this._logger.error('error getting healthcheck', err);
+                this._logger.error('error getting healthcheck', {
+                    error: err,
```

Comment:
`error: err.message`

Reply:
oops, this is my mistake, I want to pass `err` here as is because its a unique object returned from the Healthcheck class. I'll look into figuring out a better way of implementing this in a future refactor